public/components/visualizations/charts/shared/common.ts

```diff
@@ -22,30 +26,40 @@ export const getCompleteTimespanKey = (span: DimensionSpan) => {
  * @returns traces.
```

Comment:
update doc as well for adding `visTypeMetaData`? also unrelated to this PR but on the line above variable name is still `param1` placeholder

Reply:
Will address here.